packages/block-library/src/categories/edit.js

```diff
@@ -12,21 +12,35 @@ import {
 	Spinner,
 	ToggleControl,
 	VisuallyHidden,
+	ToolbarGroup,
+	ToolbarButton,
 } from '@wordpress/components';
 import { useInstanceId } from '@wordpress/compose';
-import { InspectorControls, useBlockProps } from '@wordpress/block-editor';
+import {
+	BlockControls,
+	InspectorControls,
+	useBlockProps,
+	RichText,
+} from '@wordpress/block-editor';
 import { decodeEntities } from '@wordpress/html-entities';
 import { __ } from '@wordpress/i18n';
 import { pin } from '@wordpress/icons';
 import { useEntityRecords } from '@wordpress/core-data';
 
+/**
+ * Internal dependencies
+ */
+import { toggleLabel } from '../search/icons';
```

Comment:
I'm not sure importing an icon from another block is desirable. Ideally, shouldn't all icons be from the icons package?

Reply:
I'm not sure either, perhaps it did not make sense to move it to the icons package until it was used in more than one block.

Reply:
This icon will only be used if the option stays in the toolbar. So if the option is moved to the sidebar like for the Archive block, this will be removed.